### PR TITLE
Feature/mac os saves directory correct fix

### PR DESCRIPTION
**Pull request by apodrugin**
Add correct fix for 'missed saves directory' error (I have to add platform dependent code to get 'Documents' directory, unfortunately there is no generic way to do that). Remove unnecessary scripts. Update README.md.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -69,6 +73,10 @@ else()
 	add_definitions(-Wno-reorder -pedantic-errors -Ofast -fno-fast-math)
 endif()
 
+if(MAC_OS)
```
Comment (skrax) [inline CMakeLists.txt:76]: For now this is okay, but in the future we want to move away from piling onto a sourcelist.
Perhaps you could add a CMake target instead?

Comment (apodrugin) [reply CMakeLists.txt:76]: I am not familiar with CMake. Will investigate how to do that.

Comment (apodrugin) [reply CMakeLists.txt:76]: By the way, can you recommend some CMake tutorial?

Comment (skrax) [reply CMakeLists.txt:76]: Checkout CppCon talks on CMake.
I can also recommend this: https://cliutils.gitlab.io/modern-cmake/

Comment (apodrugin) [reply CMakeLists.txt:76]: Thanks a lot

**Comment by apodrugin** [comment]
Fixed conflicts and build on Mac OS. Please, take a look.

File: CMakeLists.txt [hunk 2]
```diff
@@ -35,24 +39,47 @@ find_package(Threads REQUIRED)
...
 # swresample
 find_package(Libswresample REQUIRED)
 add_library(SWRESAMPLE::SWRESAMPLE UNKNOWN IMPORTED)
 set_target_properties(SWRESAMPLE::SWRESAMPLE PROPERTIES
   IMPORTED_LOCATION ${LIBSWRESAMPLE_LIBRARIES}
   INTERFACE_INCLUDE_DIRECTORIES ${LIBSWRESAMPLE_INCLUDE_DIRS})
+
+# MacOSUtils
+if(MAC_OS)
+  add_library(MacOSUtils macOS/src/PathUtils.m)
```
Comment (atmaxinger) [inline CMakeLists.txt]: The directory is called `macos` in lowercase, so this line breaks the build on Linux

**Comment by apodrugin** [comment]
Now it works as intended. Sorry for the mess with commits.

**Pull request merged**